Software-Engineering/L5-Principal-Software-Engineer.md

```diff
@@ -1,97 +1,124 @@
+- I worked with teams to manage multiple conflicting priorities, navigated difficult tradeoffs, and helped sequence work resulting in the best collective outcome.
 - I collaborated with another principal engineer to ensure that a feature I was helping a team with would support a feature I knew another team was planning.
+- I developed a section of the Octopus target architecture, which required a thorough understanding of the problem space, proposing potential solutions, and offering a strong and informed opinion on the best solution.
+- I contributed to building blocks or core technologies used by a variety of teams.
 
 </details>
 
 ## 🌱 Teaching and Mentoring
 
 - I am an expert mentor and teacher.
+- I am curious, listen, and encourage healthy debate.
+- I allow myself to be persuaded by great ideas.
 - I create clarity by communicating complex things in simple and accessible ways.
 - I understand my broadest impact comes from empowering those around me.
 - I am an expert at designing and reviewing software architecture and code.
-- I help teams and individuals understand our technical strategy, and the things we are doing to progress it.
+- I help teams and individuals understand our technical strategy, how it connects to our product strategy, and the things we are doing to progress it.
 - I regularly provide training and support for new methods, tools, and patterns.
-- I collaborate with managers to identify and find ways to fill any gaps in technical skills required in my teams.
-- I am curious, listen, and encourage debate.
-- I allow myself to be persuaded by great ideas.
+- I actively share knowledge relevant to multiple teams.
+- I collaborate with managers and lead software engineers to identify and find ways to fill any gaps in technical skills required in my teams.
 
 <details>
 <summary>Examples</summary>
 
 - I was consistently in demand for design and code review.
-- I am actively mentoring multiple Octopus engineers.
+- I became aware that I was a bottleneck and worked proactively with managers to uplift and empower other people to make the team more effective.
+- I am actively mentoring multiple Octopus engineers, including a Lead Software Engineer aspiring to become a Principal Software Engineer.
 - Engineers at Octopus seek out to work on my teams to learn from me.
 - After reviewing a code contribution from an engineer I saw could be improved, I worked directly with that engineer and their manager or team leader to uplift their skills in the identified growth area.
 - I helped a team make a course correction based on new information or ideas.
 - I provided appropriate technical freedom, and framing, that allowed the Lead Software Engineers on my teams to thrive independently.
+- I have elevated/shifted my focus significantly because the team(s) I support have been uplifted and empowered to the point they can deliver their mission with me as a sounding board.
```

Comment:
```suggestion
- I have elevated my focus significantly because the teams I support have been uplifted and empowered to the point they can deliver their mission with me as a sounding board.
```